ocaml/testsuite/tests/shape-index/index_aliases.reference

```diff
@@ -1,17 +1,34 @@
 Indexed shapes:
```

Comment:
If comparing this file to upstream, you'll notice that the line numbers are off. This is expected; the commend at the start of `index_aliases.ml` is formatted differently between the two files, which is the cause for the difference.